### PR TITLE
Update README.Linux.md

### PR DESCRIPTION
**Pull request by tbmreza**
debian install command

### CODE REVIEW

File: OMCompiler/README.Linux.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Then update and install OpenModelica build dependencies:
 ```bash
 sudo apt-get update
 sudo apt-get build-dep openmodelica
+sudo apt-get install openmodelica
```
Comment (AnHeuermann) [inline OMCompiler/README.Linux.md:70]: This readme is aimed at people that want to *build* OpenModelica from the sources hosted in this GitHub repository. A explanation on how to *install* OpenModelica can be found over at our [homepage](https://openmodelica.org/download/download-linux/).

Why do you need to install OpenModelica here?

**Review by AnHeuermann — COMMENTED** [review]
Clarify what's the issue you are trying to solve.

File: OMCompiler/README.Linux.md (hunk 1 above)
Comment (tbmreza) [reply OMCompiler/README.Linux.md:70]: oops sorry I was upset I found the steps anticlimax. closing the pull request.

Comment (AnHeuermann) [reply OMCompiler/README.Linux.md:70]: No worries. But if you find something to be unclear feel free to open issues about it it. Our documentation can be a bit ... challenging 😆

**Pull request closed without merging**